src/Avalonia.Base/Platform/Internal/AssemblyDescriptor.cs

```diff
@@ -3,11 +3,13 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using Avalonia.Metadata;
 using Avalonia.Utilities;
 
 namespace Avalonia.Platform.Internal;
 
-internal interface IAssemblyDescriptor
+[Unstable]
+public interface IAssemblyDescriptor
```

Comment:
These interfaces should be still internal. They always have been internal.

Reply:
I looked at the codebase and it seems members marked with unstable are still public. They will be hidden in the ref assembly so it should be fine.